frontend/src/features/idir/search/table/CompanySearchResultColumnDef.tsx

```diff
@@ -51,7 +47,7 @@ export const CompanySearchResultColumnDef: MRT_ColumnDef<CompanyProfile>[] = [
         <>
           {mailingAddress?.addressLine1}<br />
```

Comment:
Perhaps should use `getDefaultRequiredVal("", mailingAddress?.addressLine1)` here

Reply:
Perhaps for all the nullable strings to have a default value of "" for country, province, city, postal code?

Reply:
I chose to set all their defaults to `null` - it appears to be the only way to handle the conditional logic:
- am I valid? then display
- if I am valid then append a `<br />`

Reply:
The empty string doesn't work?